node-backend/app/controllers/CoreValueController.js

```diff
@@ -0,0 +1,151 @@
+const db = require("../models/sequelize");
+const CoreValue = db.core_value;
+const yup = require("yup");
+
+module.exports.create = (req, res) => {
+  //validation schema
+  const schema = yup.object().shape({
+    org_id: yup.number().required(),
+    core_value_text: yup.string().required(),
+    description: yup.string().required(),
+    parent_core_value_id: yup.number().required(),
+  });
+  // Create a core value object
+  const corevalue = {
+    org_id: req.params.organisation_id,
+    core_value_text: req.body.core_value_text,
+    description: req.body.description,
+    parent_core_value_id: req.body.parent_core_value_id,
+  };
+
+  schema.isValid(corevalue).then(function (valid) {
+    if (valid) {
+      // Save corevalue in the database
+      CoreValue.create(corevalue)
+        .then((data) => {
+          res.send(data);
+        })
+        .catch((err) => {
+          res.status(500).send({
+            message:
+              err.message ||
+              "Some error occurred while creating the Organization.",
+          });
+        });
+    } else {
+      res.send({
+        status: 422,
+        message: "Contents not be correct format",
```

Comment:
If parameters are invalid it should be 400 bad request and message should provide which parameter values are incorrect